src/components/SettlementButton.js

```diff
@@ -106,14 +109,11 @@ class SettlementButton extends React.Component {
                 }
 
                 this.setState(prevState => ({
-                    buttonOptions: [...prevState.buttonOptions.slice(0, 1),
-                        {
-                            text: this.props.translate('iou.settleVenmo'),
-                            icon: Expensicons.Venmo,
-                            value: CONST.IOU.PAYMENT_TYPE.VENMO,
-                        },
-                        ...prevState.buttonOptions.slice(1),
-                    ],
+                    buttonOptions: [...prevState.buttonOptions.splice(1, 0, {
+                        text: this.props.translate('iou.settleVenmo'),
+                        icon: Expensicons.Venmo,
+                        value: CONST.IOU.PAYMENT_TYPE.VENMO,
+                    })],
```

Comment:
**question:** What is this related to?

Reply:
It's not really related – the prop was missing and I found the existing code confusing. Going to move this to a separate PR.

Reply:
Removed this unrelated change.